cases/signals.py

```diff
@@ -5,7 +5,17 @@
 
 
 @receiver(post_save, sender=Complaint)
+def update_case_for_complaint(sender, instance, **kwargs):
+    # Update the case last modified
+    instance.case.save()
+
+
 @receiver(post_save, sender=Action)
-def update_case(sender, instance, **kwargs):
+def update_case_for_action(sender, instance, **kwargs):
+    # Action took place before the case was last
+    # modified so don't update it
+    if instance.time < instance.case.modified:
+        return
```

Comment:
Possible approach the issue of cases popping up to the top of the list when old actions are logged.

Reply:
Yes, I think this works? Trying to think of an edge case where the case is saved whilst an action is being made - e.g. a new complaint comes in, I guess, but that'd bring it up to the top anyway.